src/ApiCompat/NamespaceRemappingComparer.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
We should update all the copyrights to match what @stephentoub is doing right now. https://github.com/dotnet/buildtools/pull/418